documentation/modules/auxiliary/scanner/db2/discovery.md

```diff
@@ -0,0 +1,28 @@
+
+## About
+
+This module simply queries the DB2 discovery service for information.
+The discovery service is integrated with the Configuration Assistant and the DB2® administration server.
+Using the discovery method, catalog information for a remote server can be automatically generated in the local database and node directory.
+
+## Verification Steps
+
+To test this module, you must make sure there is at least one reacheable DB2 Discovery Service at the target address range or CIDR identifier.
+1. `use auxiliary/scanner/db2/discovery`
+2. `set RHOSTS [target address range/cidr]`
+3. `set THREDS [number of threads]`
+4. `run`
+
+
+## Scenarios
+
+```
```

Comment:
any additional info on this server, like version or host OS?  While those may not make a difference on this module, we've been adding that for consistency. Something like
```### DB2 version a.b.c.d on Ubuntu 16.04```

Reply:
This module currently returns only the node name and product id. I'll check if I can make an PR on the module to include more info on the server.

Reply:
I've checked the module implementation and it looks like we cannot get more info on the server only by the db2 discovery service

Reply:
Yes, however I assume you own the device being scanned and would therefore have some insight into what version and OS it's being run as.
While it may not be displayed *by* the module, think of it like this.  In 2 years when this module stops working, we'll look back at the tickets and docs and be able to say "we know there was good output when running on foo operating system and baz db2 version.  Lets go back to that and then confirm it still good, and that it was db2 changing how it works, and not some downstream effect of another framework change."
It's not required, but I think it makes a nice difference.

Reply:
Oh, sorry for the misunderstanding. Just added a few more details on the target.